web/web.go

```diff
@@ -461,7 +459,12 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 	tableInterleaveIssues := parentTableHelper(table, update)
```

Comment:
I was a bit stumbled on variable name `tableInterleaveIssues` since this refers to `struct TableInterleaveStatus`; should we just rename it to say `tableInterleaveStatus`?

Reply:
done